src/pkg/podman/podman.go

```diff
@@ -227,9 +227,18 @@ func IsToolboxImage(image string) (bool, error) {
 }
 
 // Pull pulls an image
-func Pull(imageName string) error {
+//
+// authfile is a path to a JSON authentication file and is internally used only
+// if it is not an empty string. The file at the path should exist
+//
+// see podman-pull(1) manual page for more information
```

Comment:
Isn't this a bit redundant? :)

Anybody who is reading the code will know that `podman pull` is involved, etc..

Reply:
True. Fixed.